package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bufferapp/components",
-  "version": "0.5.16",
+  "version": "0.5.17-beta.2",
```

Comment:
Preference here is to keep these out of PRs and do them as a fast followup on master branch (with a CHANGELOG.md change)